salesforce/modules/rest/client.bal

```diff
@@ -0,0 +1,347 @@
+    @display {label: "SOSL Search"}
+    isolated remote function search(@display {label: "SOSL Search Query"} string sosl, typedesc<record {}> returnType = <>)
+                                    returns @display {label: "SOSL Result"} stream<returnType, error?>|error = @java:Method {
+        'class: "io.ballerinax.salesforce.ReadOperationExecutor",
+        name: "searchSOSLString"
+    } external;
+
+    private isolated function processSearchSOSLString(typedesc<record {}> returnType, string searchString)
+                                                    returns stream<record {}, error?>|error {
+        string path = utils:prepareQueryUrl([API_BASE_PATH, SEARCH], [Q], [searchString]);
+        SOSLSearchResult objectInstance = check new (self.salesforceClient, path);
+        stream<record {}, error?> finalStream = new (objectInstance);
+        return finalStream;
+    }
+
+    //Describe SObjects
+    # Lists the available objects and their metadata for your organization and available to the logged-in user.
+    #
+    # + return - `OrgMetadata` record if successful or else `sfdc:Error`
+    @display {label: "Get Available Objects"}
+    isolated remote function describeAvailableObjects() 
+                                                      returns @display {label: "Organization Metadata"} 
+                                                      OrgMetadata|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Describes the individual metadata for the specified object.
+    #
+    # + sobjectName - sObject name
+    # + return - `SObjectBasicInfo` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Basic Information"}
+    isolated remote function getSObjectBasicInfo(@display {label: "sObject Name"} string sobjectName) 
+                                                 returns @display {label: "sObject Basic Information"} 
+                                                 SObjectBasicInfo|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, sobjectName]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Completely describes the individual metadata at all levels for the specified object. Can be used to retrieve
+    # the fields, URLs, and child relationships.
+    #
+    # + sObjectName - sObject name value
+    # + return - `SObjectMetaData` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Description"}
+    isolated remote function describeSObject(@display {label: "sObject Name"} string sObjectName) 
+                                             returns @display {label: "sObject Metadata"} SObjectMetaData|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, sObjectName, DESCRIBE]);
+          return check self.salesforceClient->get(path);
```

Comment:
Formatting issue

Reply:
Fixed in 8bfb122ccf270a0868c40a7ea6ddaef1d3a79a39